config.example.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * This is the main config for your NFT sale.
+ *
+ * Fill this out with all your project details.
+ */
+
+module.exports = {
```

Comment:
Shouldn't the Pinata config examples be added here? And should make a note that they are optional if you want to use Pinata.

Reply:
Oops, was in my main config file. :sweat_smile: